tests/reg_issue1698/reg_issue1698.fz.expected_err_c

```diff
@@ -1 +1 @@
-*** failed pre-condition on call to '(option error).get'
```

Comment:
The context will be available again, when we have stacktrace support?

Reply:
Yes, two things that I plan here: 
- First, the source code position and text of the failed condition should be shown, i.e., instead of just `:` I would like to see `xyz.fz:line:col: debug: condition xyz`
- Then, there should be a Fuzion-level API to obtain a stack trace